spec/helpers/basespace_helper_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "rails_helper"
+require "webmock/rspec"
+
+RSpec.describe BasespaceHelper, type: :helper do
+  describe "#files_for_basespace_dataset" do
+    context "#basespace API call returns successfully" do
+      before do
+        allow(HttpHelper).to receive(:get_json)
+          .and_return(
+            "Items" => [
+              {
+                "Name" => "Item 1",
+                "HrefContent" => "Download Path 1",
```

Comment:
I suggest using fake values that resemble the actual data, because that helps as a documentation. For example: `"HrefContent" => "v1pre3/projects/1234"` or something alike

Reply:
👍